Automate/Tests/Speckle.Automate.Sdk.Tests.Integration/GetAutomationStatus.cs

```diff
@@ -3,34 +3,34 @@
 
 namespace Speckle.Automate.Sdk.Tests.Integration;
 
-public class FunctionRun
+public struct FunctionRun
 {
   public string StatusMessage { get; set; }
 }
 
-public class AutomationRun
+public struct AutomationRun
 {
   public string Status { get; set; }
   public IList<FunctionRun> FunctionRuns { get; set; }
 }
 
-public class AutomationStatus
+public struct AutomationStatus
 {
   public string Status { get; set; }
   public IList<AutomationRun> AutomationRuns { get; set; }
 }
 
-public class ModelAutomationStatus
+public struct ModelAutomationStatus
 {
   public AutomationStatus AutomationStatus { get; set; }
 }
 
-public class ProjectAutomationStatus
+public struct ProjectAutomationStatus
 {
   public ModelAutomationStatus Model { get; set; }
 }
 
-public class AutomationStatusResponseModel
+public struct AutomationStatusResponseModel
 {
   public ProjectAutomationStatus Project { get; set; }
 }
```

Comment:
Out of scope of this PR but this seems rather odd to have in the Integration tests instead of the SDK project 🤔

Reply:
Resolving this conversation in favour of the JIRA ticket https://spockle.atlassian.net/browse/CNX-8690